package.json

```diff
@@ -101,7 +101,7 @@
     "api": "run-p check-env-file api-watch",
     "check-env-file": "npx ts-node src/checkEnvFile.ts",
     "api-watch": "nodemon -e \".ts\" --exec npm run babel-node-api",
-    "babel-node-api": "npx kill-port 6540 && babel-node -r dotenv/config -x \".ts\" ./src/api/api.ts",
```

Comment:
In my testing, this does not seem to be necessary when running `npm run dev`. It comes in handy when running the api and web app separately, but that does not seem to be a common workflow on the team.